db/migrate/20230330224511_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.integer :commentscounter
+      t.integer :likescounter
+
+      t.timestamps
```

Comment:
- [ ] As requested by the last code reviewer, please 🙏🏽 create a reference for the author_id in this file as this will create author_id and a corresponding index. 👇🏽 

`t.references :author, foreign_key: { to_table: 'users' }`

You didn't add the reference to the user table, but changed the name of a column "author_id".

Reply:
## I'm sorry @maaloum I didn0t find this document 👇🏽 

![image](https://user-images.githubusercontent.com/58642949/228999384-473a867c-69d8-4f7e-ad2e-33e9babb80ae.png)

# ⚠️ I hope the next CR just approves your PR. ⚠️